bindings_generator/Cargo.toml

```diff
@@ -3,7 +3,5 @@ name = "bindings_generator"
 version = "0.1.0"
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
 bindgen = "0.65.1"
```

Comment:
Trailing newline :)

Reply:
Added an extra line at the end of the file in latest commit

Reply:
Also this file, now only the root file was updated. Same for `ci.yml` :)